client/src/components/AssessmentCard/AssessmentCard.tsx

```diff
@@ -19,83 +19,65 @@ const formatDate = (date: Date): string =>
     hour12: false,
   });
 
+const getIcon = (name: string): string => {
+  // If assessment name contains any of the keywords, use the star icon, otherwise use the event icon
+  return name
+    .toLowerCase()
+    .split(" ")
+    .some((word) =>
```

Comment:
```suggestion
    .toLowerCase()
    .replace(/[^a-z0-9 ]/g, "")
    .split(" ")
    .some((word) =>
```
(for example "Test!")